lxc/cluster.go

```diff
@@ -248,10 +251,31 @@ func (c *cmdClusterRemove) Command() *cobra.Command {
 
 	cmd.RunE = c.Run
 	cmd.Flags().BoolVarP(&c.flagForce, "force", "f", false, i18n.G("Force removing a member, even if degraded"))
+	cmd.Flags().BoolVarP(&c.flagNonInteractive, "quiet", "q", false, i18n.G("Don't require user confirmation for using --force"))
 
 	return cmd
 }
 
+func (c *cmdClusterRemove) promptConfirmation(name string) error {
+	reader := bufio.NewReader(os.Stdin)
+	fmt.Printf(i18n.G(`After a server is removed from the cluster, it won't be able to manage
+anymore the containers, images or custom volumes it might currently
+have on it.
```

Comment:
Wording nit, how about we say:

```
Forcefully removing a server from the cluster should only be done as a last resort.
The removed server will not be functional after this action and will require a full reset of LXD, loosing any remaining container, image or storage volume that the server may have held.

When possible, an online removal should be preferred, this will require you move any affected container, image or storage volume to another server prior to the server being cleanly removed from the cluster.

The --force flag should only be used if the server has died, been reinstalled or is otherwise never expected to come back up.
```

Reply:
Changed.